downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md

```diff
@@ -30,6 +30,40 @@ If you have not installed Ballerina, download the [installers](/downloads/#swanl
 
 ## Backward-incompatible changes
 
+- A bug that allowed assigning `anydata` value to a `readonly` `anydata` variable has been fixed.
+
+    ```ballerina
+    public function main() {
+        anydata a = [];
+        anydata & readonly b = a; // Compilation error now.
+    }
+    ```
+  
+- A bug that allowed including a `readonly` `service` object inside a non-`service` `readonly` class has been fixed.
+
+    ```ballerina
+    public type A readonly & service object {
+        isolated remote function execute() returns int;
+    };
+  
+    readonly client class B {
+        *A;
+  
+        isolated remote function execute() returns int {
+            return 2;
+        }
+    }
+    ```
+  
+- Modified the behavior of the [runtime Java APIs to support the intersection type](/downloads/swan-lake-release-notes/swan-lake-2201.8.0#intersection-type-support-in-runtime-java-apis).
+
+- `TypeSymbol` of an intersection type will now return an `IntersectionTypeSymbol`.
```

Comment:
```suggestion
- `typeDescriptor()` of symbols of intersection-type now returns the type-symbol as of `IntersectionTypeSymbol` instead of its effective-type
```